charts/stable/common/templates/lib/controller/_volumemounts.tpl

```diff
@@ -0,0 +1,49 @@
+{{/*
+Volumes included by the controller.
+*/}}
+{{- define "common.controller.volumeMounts" -}}
+
+{{- range $index, $PVC := .Values.persistence }}
+{{- if $PVC.enabled }}
+- mountPath: {{ $PVC.mountPath | default (printf "/%v" $index) }}
+  name: {{ $index }}
+{{- if $PVC.subPath }}
+  subPath: {{ $PVC.subPath }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{- if .Values.additionalVolumeMounts }}
+  {{- toYaml .Values.additionalVolumeMounts | nindent 2 }}
```

Comment:
I _think_ this should be indented to 0 now? to put it on the same indent level as the other volumeMounts